validator.py

```diff
@@ -553,6 +553,48 @@ def validate_dependencies(self):
         self.validate_events()
         self.validate_messages()
         self.are_all_scenes_reachable()
+        self.validate_quantized_support()
+
+
+    def validate_quantized_support(self):
+        '''
+        Flag if treatments_required > 1 for unsupported injures.
+        In general, these are injuries that aren't successfully treated by hemostatic gauze or pressure bandage.
+        '''
+        data = copy.deepcopy(self.loaded_yaml)
+        self.validate_quantized_support_in_characters(data['state']['characters'])
+
+        for scene in data['scenes']:
+            if 'state' in scene and 'characters' in scene['state']:
+                self.validate_quantized_support_in_characters(scene['state']['characters'])
+
+
+    def validate_quantized_support_in_characters(self, characters):
+        for character in characters:
+            for injury in character.get('injuries', []):
+                if 'treatments_required' in injury:
+                    required = injury['treatments_required']
+                    type = injury['name']
+                    location = injury['location']
+                    # look for an injury type that doesn't support quantized injuries
+                    if required > 1 and not self.supports_quantized_injury(type, location):
+                        self.logger.log(LogLevel.ERROR, f"Injuries requiring multiple treatments are only supported when the injury is treated by hemostatic gauze or pressure bandage, but not '{type}' injuries at '{location}' location in character '{character['id']}'.")
+                        self.invalid_values += 1
+
+
+    def supports_quantized_injury(self, injury_type, location):
+        '''
+        Returns whether the specified injury/location combination is successfully treated by a hemostatic gauze or pressure bandage.
+        '''
+        if injury_type == 'Laceration' and 'thigh' in location:
+            return False # takes a Tourniquet
+        if injury_type == 'Shrapnel' and 'face' in location:
+            return False # untreatable
+        if injury_type == 'Puncture' and 'bicep' in location or 'thigh' in location or 'calf' in location or 'chest' in location:
```

Comment:
Can you add parentheses here just to make it more clear to the reader x and (y or z or w)?